scene/3d/visual_instance.cpp

```diff
@@ -180,6 +180,22 @@ bool VisualInstance::get_layer_mask_bit(int p_layer) const {
 	return (layers & (1 << p_layer));
 }
 
+void VisualInstance::set_sorting_offset(float p_offset) {
+	sorting_offset = p_offset;
+	VisualServer::get_singleton()->instance_set_pivot_data(instance, sorting_offset, sorting_use_aabb_center);
+}
+float VisualInstance::get_sorting_offset() {
```

Comment:
```suggestion

float VisualInstance::get_sorting_offset() {
```
While in the header you can group methods like that, in cpp we prefer leaving a space between each method.